command/commands/team.py

```diff
@@ -43,9 +43,10 @@ class TeamCommand:
            "    * permanently delete the specified team\n"
     desc = "for dealing with " + command_name + "s"
 
-    def __init__(self):
-        """Initialize team command parser."""
+    def __init__(self, sc):
+        """Initialize team command parser with given Slack Client Interface."""
```

Comment:
For future reference, we may want to move parameter documentation to a `:param:` field in the docstring - I think that's missing in many places though, so I wouldn't change it now.